lib/routes/ride/routes.js

```diff
@@ -0,0 +1,61 @@
+import { createRide, getRideById } from '@daos/ridesDao';
+import Joi from 'joi';
+
+export default [
+  {
+    method: 'POST',
+    path: '/start',
+    options: {
+      description: 'driver starts the ride',
+      notes: 'driver starts the ride',
+      tags: ['api', 'driver-start-ride'],
+      cors: true,
+      validate: {
+        payload: Joi.object({
+          userId: Joi.string(),
+          cabId: Joi.string(),
+          startPoint: Joi.object({
+            latitude: Joi.number(),
+            longitude: Joi.number(),
+          }),
+        }),
+      },
+    },
```

Comment:
Abstract the validation out

For example rideStart validation. If it is a huge object then you are adding too much in the route. So abstracting will help here